.github/workflows/deploy-book.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     - name: Publish the book
       run: |
-        nix-build -A docs
+        nix build .#docs
```

Comment:
While we are in here, maybe just steal https://github.com/goodbyekansas/nedryglot/blob/main/.github/workflows/deploy-book.yml